examples/zkapps/06-offchain-storage/offchain-storage-zkapp/ui/pages/index.page.tsx

```diff
@@ -0,0 +1,303 @@
+import React, { useEffect, useState } from 'react';
+import './reactCOIServiceWorker';
+
```

Comment:
Suggestion:  import styles and GradientBG to use in UI. See other comment below in file to see what it looks like. 
```
import styles from '../styles/Home.module.css';
import GradientBG from '../components/GradientBG';
```